### DISCUSSION TITLE
How to create a custom bestiary?

### DISCUSSION THREAD

**Discussion by cmcknight** [Obsidian-TTRPG-Community/fantasy-statblocks#119] [Q&A]
I would like to create a custom bestiary for the statblock plugin for my game. Where would I store the file (I'm assuming I can select the appropriate file from the settings)?

**Comment by valentine195** [comment]
What do you mean by a bestiary? 

The plugin has an internal bestiary that you can add to. You can do this with notes (using frontmatter), by creating Statblocks in notes and saving them, or by creating creatures directly in settings.

**Reply by valentine195** [reply]
See here https://github.com/valentine195/obsidian-5e-statblocks#the-bestiary

**Comment by cmcknight** [comment]
I’m running an Esper Genesis game, so I wanted to create a Bestiary file for it. I’m looking at the settings on the plugin, and it looked like there was a way to do a bulk import from JSON, but I may not be understanding what is going on under the covers. :-)

**Reply by valentine195** [reply]
The bulk imports are mostly geared towards 5e specific properties. If you are programming-inclined you could make your own transform and add them to the plugin’s data file.

**Comment by cmcknight** [comment]
I am programmatically inclined, but Esper Genesis is a 5e system so I was looking at just doing a JSON file and a bulk import. However, since I have to enter the data anyway, I can just do onesies as I enter it. Would you recommend setting the stats in the YAML frontmatter or in a statblock in the body?

**Comment by valentine195** [comment]
Frontmatter is more useful long term imo
